psi4/src/psi4/libfock/jk.h

```diff
@@ -52,6 +54,8 @@ namespace pk {
 class PKManager;
 }
 
+typedef unsigned long long int eri_index;
```

Comment:
Do we really need an integer type this large?

Reply:
Not sure yet, I just made it arbitrarily big to be safe. @davpoolechem Thoughts? Change to `size_t` instead? Needs to fit nshell^4.

Reply:
Andy and I discussed this. We likely will need an integer type this large, since eri_index goes up to nshell^4, and nshell can go up to thousands of shells for larger systems.

Reply:
Isn't `size_t` designed for exactly this purpose, an unsigned type that can address all of the reachable memory?

Reply:
Mainly, we wanted to _guarantee_ that eri_index would be an unsigned integer of at least 64 bits. Size_t would achieve this guarantee for 64-bit systems, but not necessarily on 32-bit systems. So I guess use of unsigned long long over size_t is a concession to the people running on 32-bit systems more than anything. Whether that's a concession we should make or not is an issue that I am neutral on.

I will leave the ball in Andy's court as to how he wants to handle this.

Reply:
I will change to `size_t`, for consistency with the rest of Psi4.